### ISSUE TITLE
Initial crc start stops at "Starting network time synchronization in CodeReady Containers VM"

### ISSUE THREAD

**Issue by sandervanvugt** [code-ready/crc#1805]
### General information

  * OS: Linux / Fedora 33 Workstation
  * Hypervisor: KVM 
  * Did you run `crc setup` before starting it (Yes)?
  * Running CRC on: Laptop /  VM (issues occurs in FC33 directly on laptop hardware as well as in KVM virtual machine)

## CRC version
```bash
CodeReady Containers version: 1.20.0+ef3f80d
OpenShift version: 4.6.6 (embedded in executable)
```
  
## CRC status
```bash
[sander@ysl ~]$ crc status --log-level debug
DEBU CodeReady Containers version: 1.20.0+ef3f80d 
DEBU OpenShift version: 4.6.6 (embedded in executable) 
Checking file: /home/sander/.crc/machines/crc/.crc-exist
DEBU About to run SSH command:
df -B1 --output=size,used,target /sysroot | tail -1 
DEBU SSH command results: err: <nil>, output: 32720793600 14361313280 /sysroot 
DEBU About to run SSH command with hidden output  
DEBU SSH command failed                           
DEBU cannot get OpenShift status: ssh command error:
command : oc get co -ojson --context admin --cluster crc --kubeconfig /opt/kubeconfig
err     : Process exited with status 1
output  :  
CRC VM:          Running
OpenShift:       Not Reachable (v4.6.6)
Disk Usage:      14.36GB of 32.72GB (Inside the CRC VM)
Cache Usage:     13.69GB
Cache Directory: /home/sander/.crc/cache
```

## CRC config
```bash
[sander@ysl ~]$ crc config view
[sander@ysl ~]$ ```
### There is no config view output

## Host Operating System
```bash
[sander@ysl ~]$ cat /etc/os-release
NAME=Fedora
VERSION="33 (Workstation Edition)"
ID=fedora
VERSION_ID=33
VERSION_CODENAME=""
PLATFORM_ID="platform:f33"
PRETTY_NAME="Fedora 33 (Workstation Edition)"
ANSI_COLOR="0;38;2;60;110;180"
LOGO=fedora-logo-icon
CPE_NAME="cpe:/o:fedoraproject:fedora:33"
HOME_URL="https://fedoraproject.org/"
DOCUMENTATION_URL="https://docs.fedoraproject.org/en-US/fedora/f33/system-administrators-guide/"
SUPPORT_URL="https://fedoraproject.org/wiki/Communicating_and_getting_help"
BUG_REPORT_URL="https://bugzilla.redhat.com/"
REDHAT_BUGZILLA_PRODUCT="Fedora"
REDHAT_BUGZILLA_PRODUCT_VERSION=33
REDHAT_SUPPORT_PRODUCT="Fedora"
REDHAT_SUPPORT_PRODUCT_VERSION=33
PRIVACY_POLICY_URL="https://fedoraproject.org/wiki/Legal:PrivacyPolicy"
VARIANT="Workstation Edition"
VARIANT_ID=workstation

```

### Steps to reproduce

  1. crc stop
  2. crc cleanup
  3. crc setup
  4. crc start -p pull-secret

### Expected
crc comes up

### Actual
crc gets stalled with "INFO starting network time synchronisation in CodeReady Containers VM"

### Logs

Before gather the logs try following if that fix your issue
```bash
$ crc delete -f
$ crc cleanup
$ crc setup
$ crc start --log-level debug
```

##Please consider posting the output of `crc start --log-level debug`  on http://gist.github.com/ and post the link in the issue.
See here: https://gist.github.com/sandervanvugt/aa9887bf254c5c058ef70db916c04273

**Comment by praveenkumar** [comment]
@sandervanvugt as per your current log gist I am not seeing any error message about `Network Time` sync, can you make sure it is full log which also consist the error message or upload a fresh one?

Below snip from the logs which you shared shows that it able ot execute the time sync command without any error.

```
INFO Starting network time synchronization in CodeReady Containers VM 
DEBU About to run SSH command:
sudo timedatectl set-ntp on 
DEBU SSH command results: err: <nil>
```

**Comment by sandervanvugt** [comment]
I repeated the procedure. Next gist is here: https://gist.github.com/sandervanvugt/7bce42718d9a09d1a7ea0708d3059f68
You will find the crc start --log-level debug shows the same result as in my previous gist. I have also included the output of a regular crc start, so that you can see that without the --log-level debug it shows "Starting time synchronisation in CodeReady Containers VM". But, as you've mentioned, while using --log-level debug it seems to be coming beyond this point, and it's getting stalled on the "About to run SSH command: host -R 3 foo.apps-crc.testing"

**Comment by gbraad** [comment]
> "About to run SSH command: host -R 3 foo.apps-crc.testing"

That sounds like https://github.com/code-ready/crc/issues/1790, but a different cause?
Can we skip this check and see if this would work?

**Comment by bczoma** [comment]
similar issue for
```
$ crc version
CodeReady Containers version: 1.21.0+68a4cdd7
OpenShift version: 4.6.9 (embedded in executable)
```

**Comment by cfergeau** [comment]
By "similar issue", you mean you are on a linux system, and `crc start` hangs at `About to run SSH command: host -R 3 foo.apps-crc.testing`? If not, can you open a new issue providing the requested information? thanks!

**Comment by bczoma** [comment]
> By "similar issue", you mean you are also on a linux system, and `crc start` hangs at `About to run SSH command: host -R 3 foo.apps-crc.testing`?

Yes, this was the case.

**Comment by cfergeau** [comment]
https://github.com/code-ready/crc/issues/1805#issuecomment-753251617

If you ctrl+c once it's hanged, could you try running:
```
ssh -i ~/.crc/machines/crc/id_ecdsa -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null  core@192.168.130.11 host -R 3 foo.apps-crc.testing
```
and 
```
ssh -i ~/.crc/machines/crc/id_ecdsa -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null  core@192.168.130.11  sudo podman ps -a
```
